hatch/src/cli/commands/new.rs

```diff
@@ -119,12 +116,14 @@ impl<'command> Command<'command> for New {
       // create the hatch_modules directory inside the project directory
       fs::create_dir(modules_path(&dir_path))?;
 
-      let deps: Vec<Dependency> = deps_from_cli.iter().map(|repo| {
+      let deps = deps_from_cli.iter().map(|repo| {
         Dependency::new(repo.1.clone(), repo.0.clone())
-      }).collect();
-      
-      clone_project_deps(modules_path(&dir_path).as_path(), &deps)?;
-      
+      }).collect::<Vec<_>>();
+
+      if deps.len() > 0 {
```

Comment:
everywhere you do `deps.len()` condition check, you should replace with a call to `dep.is_empty()`

Reply:
Agreed